contentcuration/contentcuration/dev_settings.py

```diff
@@ -10,9 +10,6 @@
 def custom_show_toolbar(request):
     return not any(request.path.startswith(url) for url in EXCLUDED_DEBUG_URLS)
 
-# TODO: REMOVE ONCE DONE TESTING
-DEFAULT_FILE_STORAGE = 'contentcuration.utils.gcs_storage.GoogleCloudStorage'
```

Comment:
Needed to do this (together with merging migrations) to be able to run devserver, but can revert if needed later!

Reply:
Hopefully we'll get that migration taken care of soon!